packages/next/server/config-shared.ts

```diff
@@ -116,7 +116,7 @@ export type NextConfig = { [key: string]: any } & {
   httpAgentOptions?: { keepAlive?: boolean }
   future?: {
     /**
-     * @deprecated this options was moved to the top level
+     * @deprecated This option was moved to the top level
```

Comment:
```suggestion
     * @deprecated This option has been removed as webpack 5 is now default
```
We should probably add this line above the [top-level config item](https://github.com/vercel/next.js/blob/f81a6a5f0fc3a33a51112d4d3261d431e704b0da/packages/next/server/config-shared.ts#L63) as well since both have no affect now.